6_drivers/in/AOSS_solar_README.md

```diff
@@ -0,0 +1,389 @@
+**source
```

Comment:
This looks like more a .txt than a .md file. Doesn't really matter, but the .md extension is causing GitHub to display the whole thing in bold font, and it's possible there are some other formatting surprises in here...

Reply:
fixed